benchmarks/triton_kernels_benchmark/benchmark_testing.py

```diff
@@ -159,7 +159,9 @@ def extract_kernels(funcs):
     # For details: https://github.com/pytorch/pytorch/issues/144778
     kernels = [kernel for kernel in kernels if kernel != []]
     # FIXME: relaxation for new agama release
-    assert len(kernels) >= n_repeat - 1, "the profiling number not match"
+    assert len(kernels) == n_repeat - 1, (
```

Comment:
`len(kernels) == n_repeat` with current agama driver.
```suggestion
    assert len(kernels) >= n_repeat - 1, (
```